src/tests/functional/plugin/cpu/single_layer_tests/broadcast.cpp

```diff
@@ -367,6 +427,19 @@ INSTANTIATE_TEST_CASE_P(smoke_DynamicShape5D, BroadcastLayerCPUTest,
                             ::testing::Values(CommonTestUtils::DEVICE_CPU)),
                         ::testing::Values(CPUSpecificParams{{}, {}, {}, "ref"})),
                     BroadcastLayerCPUTest::getTestCaseName);
+
+INSTANTIATE_TEST_CASE_P(smoke_DynamicShape5DScalar, BroadcastLayerCPUTest,
+                    ::testing::Combine(
+                        ::testing::Combine(
+                            ::testing::ValuesIn(dynamicInputShapesScalar),
+                            ::testing::Values(std::vector<int64_t>{8, 16, 1, 1, 7}),
```

Comment:
Could you please also cover case when target_shape == {0} which is common case for dynamic models?

Reply:
Fixed. Thanks Dmitry!